ssl/ssl_lib.c

```diff
@@ -3029,6 +3049,20 @@ long ossl_ctrl_internal(SSL *s, int cmd, long larg, void *parg, int no_quic)
     }
 }
 
+const char *ssl_connection_prov_querry(const SSL_CONNECTION *ssl_conn)
+{
+    /* Return propq from SSL object prior when set,
+     * otherwise use the SSL_CTX propq.
+     */
+    if (ssl_conn != NULL) {
+        if (SSL_CONNECTION_GET_SSL(ssl_conn) != NULL)
+            return SSL_CONNECTION_GET_SSL(ssl_conn)->propq;
+        else if (SSL_CONNECTION_GET_CTX(ssl_conn) != NULL)
+            return SSL_CONNECTION_GET_CTX(ssl_conn)->propq;
+    }
```

Comment:
I do not think it does what you comment here about. `SSL_CONNECTION_GET_SSL()` basically cannot return NULL unless ssl_conn is NULL.

Also the multiline comment must start with `\*` on a separate line.